masonry/src/app/render_root.rs

```diff
@@ -339,6 +341,7 @@ impl RenderRoot {
         match event {
             WindowEvent::Rescale(scale_factor) => {
                 self.scale_factor = scale_factor;
+                self.global_state.scale_factor = scale_factor;
```

Comment:
There seems to be some redundancy there. You should probably remove `RenderRoot::scale_factor` and have `RenderRootState::scale_factor` be the only canonical value.

Reply:
removed from Root